js/index.js

```diff
@@ -0,0 +1,72 @@
+$(document).ready(function () { 
+
+	// // hide all answers/
+	$('#learnMoreText, #readMoreText, #read-less, #learn-less').addClass('hide');
+
+	$("#read-more").click(readMore);
+	$("#learn-more").click(learnMore);
+	$('#read-less').click(readLess);
+	$('#learn-less').click(learnLess);
+
+
+
+	function readMore () {
+		$("#readMoreText").slideDown();
+		$("read-more").slideDown();
+		$("#read-more").hide();
+		$("#read-less").slideDown();
+}
+
+	function readLess () {
+		$("#readMoreText").slideUp();
+		$("#read-less").slideUp();
+		$("#read-less").hide();
+		$("#read-more").show();
+
+}
+
+	function learnMore () {
+		$("#learnMoreText").slideDown();
+		$("#learn-more").slideDown();
+		$("#learn-more").hide();
+
+}
+function learnLess () {
+		$("#learnMoreText").slideUp();
+		$("#learn-less").hide();
+		$("#learn-more").show();
+
+}
+
+}
+)
```

Comment:
Usually, people write these last two lines (41 + 42) as this: `})` next to one another. Also, don't forget to close end it with a semicolon `});`